frontend/src/components/Collaboration/Chat.js

```diff
@@ -38,13 +41,24 @@ const Chat = ({ socket, roomId, user }) => {
     });
   }, []);
 
+  useEffect(() => {
+    // Scroll to the bottom of the chat container
+    chatContainerRef.current.scrollTop = chatContainerRef.current.scrollHeight;
+  }, [messages]);
+
   return (
     <div className='chat-container'>
-      <div className='chat-messages'>
+      <div className='chat-messages' ref={chatContainerRef}>
         {messages.map((msg, index) => (
           <div
             key={index}
-            className={`message ${msg.sender === user ? 'self' : 'other'}`}
+            className={`message ${
+              msg.sender === user
+                ? 'self'
+                : msg.sender === 'server'
+                ? 'server'
+                : 'other'
+            }`}
```

Comment:
IMO, a bit hard to read. Does this looks better?
```js
className={`message ${
  msg.sender === user
  ? 'self'
  : (msg.sender === 'server' ? 'server' : 'other')
}`}
```

Reply:
Changed